ios-swift/HelloWorldSwift/AppDelegate.swift

```diff
@@ -125,5 +134,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let notification:NSNotification = NSNotification(name:"message_received", object:nil, userInfo:userInfo)
         NSNotificationCenter.defaultCenter().postNotification(notification)
         println("UPS message received: \(userInfo)")
+        
+        // Send metrics when app is launched due to push notification
+        AGPushAnalytics.sendMetricsWhenAppAwoken(application.applicationState, userInfo: userInfo)
```

Comment:
Just wondering, shouldn't we implement the `application:didReceiveRemoteNotification:fetchCompletionHandler:` instead of the `application:didReceiveRemoteNotification:` ?